### PR TITLE
Widgets timeseries filtered from meta

### PR DESCRIPTION
**Pull request by rochoa**
cc @javierarce @viddo as you were working on histogram and time-series

### CODE REVIEW

File: src/geo/ui/widgets/time-series/torque-histogram-view.js [hunk 1]
```diff
@@ -80,6 +80,7 @@ module.exports = View.extend({
 
   _onBrushEnd: function(loBarIndex, hiBarIndex) {
     var data = this.model.getData();
+    this._rangeFilter.set('columnType', this._torqueLayerModel.get('meta').column_type);
```
Comment (rochoa) [inline src/geo/ui/widgets/time-series/torque-histogram-view.js]: As I pointed in the commit itself, I'm not happy with this approach. Any help to improve this is welcome.

Comment (viddo) [inline src/geo/ui/widgets/time-series/torque-histogram-view.js]: The meta prop is not available initially, right?
I think this detail is not a concern for the view but the model, which knows about both the filter and layer.

**Comment by viddo** [comment]
Overall looks good :+1: 
Can't get the examples to work locally though, so could not test it. :/

**Comment by javierarce** [comment]
:+1: I have the same issue with the examples as Nicklas, but code looks fine.

File: src/windshaft/dashboard.js [hunk 2]
```diff
@@ -56,17 +56,18 @@ WindshaftDashboard.prototype._createInstance = function(options) {
...
 
-      this._updateWidgetURLs({
-        layerId: options.layerId
-      });
-
       // update other kind of layers too
-      this.layers.each(function(layer) {
+      this.layers.each(function(layer, layerIndex) {
         if (layer.get('type') === 'torque') {
+          layer.set('meta', dashboardInstance.getLayerMeta(layerIndex));
```
Comment (viddo) [inline src/windshaft/dashboard.js]: The meta information that is required to have the column type only works for Torque for now? I guess this is related to why the default time-series widget doesn't work?

**Comment by viddo** [comment]
Btw,this resolves #880 right?

File: src/windshaft/dashboard.js (hunk 2 above)
Comment (rochoa) [inline src/windshaft/dashboard.js]: Yes, that kind of information is only provided for torque layers, and at the same time is required by time-series widget associated to torque layers.

We need to review this use case when using time-series widget with non-torque layers.

**Comment by rochoa** [comment]
@viddo Yes, I updated the example to not use the hacks.

**Comment by rochoa** [comment]
And ... :ship: ing :it: Thx @viddo for your help!

**Pull request merged**